docs/elements/hx-tile-description/index.html

```diff
@@ -0,0 +1,28 @@
+---
+title: <hx-tile-description>
+also:
+  components/choice-tiles: Choice Tiles
+  elements/hx-tile: <hx-tile>
+  elements/hx-tile-title: <hx-tile-title>
+---
+{% extends 'element.njk' %}
+{% block content %}
+  <section>
+    <p>
+      The custom <code>&lt;hx-tile-description&gt;</code> element 
```

Comment:
"The optional <hx-tile-description> element provides a semantic container for the tile's description."